src/types/api.ts

```diff
@@ -153,13 +153,65 @@ export type Shape = RectShape | PointShape;
 /**
  * Selector which identifies a region of the document defined by a shape.
  *
- * TODO:
- *  - Specify the origin used by the shape coordinates (eg. page, document body)
- *  - Specify the coordinate system
+ * # Anchors
+ *
+ * The shape's coordinates may be relative to an _anchor_ element within the
+ * document. For example a page in a PDF or an `<img>` an an HTML document. For
+ * document types where a location can be fully specified by the shape
+ * coordinates alone (such as images, but not PDFs or HTML documents), the
+ * anchor is optional.
+ *
+ * # Coordinate systems
+ *
+ * Shape selectors should be defined using the natural coordinate system for the
+ * anchor element in the document, enabling an annotation made in one viewer to
+ * be resolved to the same location in a different viewer, with different view
+ * settings (zoom, rotation etc.). For common document and anchor types, these
+ * are as follows:
+ *
+ * - For PDFs, PDF user space coordinates (points), with the origin at the
+ *   bottom-left corner of the page.
```

Comment:
I've seen in some previous PRs that we use the bottom-left corner as the origin for PDFs. Do you know if this is something PDFJS does because they follow some standard for PDFs, or is it something they decided at some point and has become a historical decision?

Reply:
This is how the "user space" coordinate system in PDF files works. There are more details in section [8.3.2.3](https://opensource.adobe.com/dc-acrobat-sdk-docs/pdfstandards/PDF32000_2008.pdf) of the PDF spec. The specification does say that the orientation can be modified on a per-page basis, in that case you might see `view.top < view.bottom`.

Reply:
Thanks!